pkg/openstack/loadbalancer.go

```diff
@@ -489,6 +492,14 @@ func getProxyProtocolFromServiceAnnotation(service *corev1.Service) *v2pools.Pro
 	}
 }
 
+func getLBMethodFromServiceAnnotation(service *corev1.Service) *v2pools.LBMethod {
+	lbMethodValue := getStringFromServiceAnnotation(service, ServiceAnnotationLoadBalancerLbMethod, "")
```

Comment:
can we use `lbaas.opts.LBMethod` as a default value?

Reply:
function removed (since we are now using a `string` instead of `v2pools.LBMethod`)